modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-web/src/main/java/com/liferay/dynamic/data/mapping/form/web/internal/display/context/DDMFormViewFormInstanceRecordsDisplayContext.java

```diff
@@ -456,6 +456,26 @@ public int getTotalItems() {
 		return searchContainer.getTotal();
 	}
 
+	public List<Long> getTotalItemsIds() {
```

Comment:
@rijema please rename it to `getDDMFormInstanceRecordIds`

Reply:
Sure. Thanks!